src/realm/sync/noinst/client_impl_base.cpp

```diff
@@ -890,6 +890,10 @@ void Connection::handle_pong_timeout()
 
 void Connection::initiate_write_message(const OutputBuffer& out, Session* sess)
 {
+    // Stop sending messages if an error was received.
```

Comment:
Can we clarify that the error we're receiving here is a websocket error rather than a sync protocol error message, since if we receive a sync protocol error message we probably will want to send an UNBIND?

Reply:
good point. done